app/packages/forms/controllers/form_results.coffee

```diff
@@ -25,3 +31,196 @@ Template.form_results.helpers
   forms: ->
     ids = Template.instance().selectedFormIds.get()
     Template.instance().data.forms.find objectId: {$in: ids}
+
+Template.form_results.onRendered ->
+  instance = @
+  @survey.getForms()
+    .then (forms) ->
+      forms.forEach (form) ->
+        _form = form.toJSON()
+        # populate the @forms minimongo collection
+        form.getTrigger()
+          .then (trigger) ->
+            _form.trigger = trigger.toJSON()
+            delete _form.triggers
+            instance.forms.upsert _form.objectId, _form
+        # get forms
+        form.getQuestions()
+          .then (questions) ->
+            _questions = questions.map (item) ->
+              item.toJSON().objectId
+            instance.forms.update _form.objectId, $set: _questions: _questions
+            questions.forEach (question) ->
+              _question = question.toJSON()
+              _question.questionId = _question.objectId
+              # populate the @questions minimongo collection
+              instance.questions.upsert _question.objectId, _question
+    .fail (err) ->
+      console.log err
+  # Participants
+  formIds = null
+  @survey.getForms()
+    .then (forms) ->
```

Comment:
We chatted about it and we'll re-factor it later if that's okay with you @Broham